config/default.ini

```diff
@@ -2,9 +2,9 @@
 [APPLICATION]
 name:          Scratch2Catrobat Converter
 short_name:    S2CC
-version:       0.8.0
+version:       0.8.1
 build_name:    Aegean cat
-build_number:  842-develop
+build_number:  1
```

Comment:
Please change the build number here.
If you pull from develop and the version number should change bach to 0.8.0 please change it back to 0.8.1